<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Table</class>
 <widget class="QWidget" name="Table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_01">
         <item>
          <widget class="QGroupBox" name="Player2">
           <property name="title">
            <string>Player 2</string>
           </property>
           <layout class="QHBoxLayout" name="player2"/>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="Player3">
           <property name="title">
            <string>Player 3</string>
           </property>
           <layout class="QHBoxLayout" name="player3"/>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_02">
         <item>
          <widget class="QGroupBox" name="Monitor">
           <property name="title">
            <string>Monitor</string>
           </property>
           <layout class="QHBoxLayout" name="monitor"/>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="EightsChooser">
           <property name="title">
            <string>Eights Chooser</string>
           </property>
           <layout class="QHBoxLayout" name="eightsChooser"/>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="QuteChooser">
           <property name="title">
            <string>Qute Chooser</string>
           </property>
           <layout class="QHBoxLayout" name="quteChooser"/>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="JpointsChooser">
           <property name="title">
            <string>Jpoints Chooser</string>
           </property>
           <layout class="QHBoxLayout" name="jpointsChooser"/>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="Played">
           <property name="title">
            <string>Played</string>
           </property>
           <layout class="QHBoxLayout" name="played"/>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="Drawn">
           <property name="title">
            <string>Drawn</string>
           </property>
           <layout class="QHBoxLayout" name="drawn"/>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_03">
         <item>
          <widget class="QGroupBox" name="Blind">
           <property name="title">
            <string>Blind</string>
           </property>
           <layout class="QHBoxLayout" name="blind"/>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="JsuitChooser">
           <property name="title">
            <string>Jsuit Chooser</string>
           </property>
           <layout class="QHBoxLayout" name="jsuitChooser"/>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="Stack">
           <property name="title">
            <string>Stack</string>
           </property>
           <layout class="QHBoxLayout" name="stack"/>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_04">
         <item>
          <widget class="QGroupBox" name="Playable">
           <property name="title">
            <string>Playable</string>
           </property>
           <layout class="QHBoxLayout" name="playable"/>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_05">
         <item>
          <widget class="QGroupBox" name="Player1">
           <property name="title">
            <string>Player 1</string>
           </property>
           <layout class="QHBoxLayout" name="player1"/>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
